Wander/app/src/main/java/com/teamwan/wander/MainMenu.java

```diff
@@ -103,4 +110,62 @@ public void setUpNotifications(){
 
         alarmManager.setRepeating(AlarmManager.RTC_WAKEUP, calendar.getTimeInMillis(), AlarmManager.INTERVAL_DAY, pendingIntent);
     }
+
+    /**
+     * Method for displaying ICA overlay and content.
+     */
+    public void initialiseICA() {
+        LinearLayout overlay = (LinearLayout)findViewById(R.id.ICAOverlay);
+        LinearLayout contentBox = (LinearLayout)findViewById(R.id.ICAContentBox);
+
+        TextView title = (TextView)findViewById(R.id.ICATitle);
+        TextView body = (TextView)findViewById(R.id.ICABody);
+        TextView accept = (TextView)findViewById(R.id.ICAAccept);
+        TextView quit = (TextView)findViewById(R.id.ICAQuit);
+
+        overlay.setVisibility(View.VISIBLE);
+        contentBox.setVisibility(View.VISIBLE);
+        title.setVisibility(View.VISIBLE);
+        body.setVisibility(View.VISIBLE);
+        accept.setVisibility(View.VISIBLE);
+        quit.setVisibility(View.VISIBLE);
+
+
+
+        Typeface tf =Typeface.createFromAsset(getAssets(),"fonts/FuturaLT.ttf");
+        title.setTypeface(tf);
+        body.setTypeface(tf);
+        accept.setTypeface(tf);
+        quit.setTypeface(tf);
+
+    }
+
+    /**
+     * Method for accepting ICA and closing overlay.
+     */
+    public void onClickICAAccept(View v) {
+        LinearLayout overlay = (LinearLayout)findViewById(R.id.ICAOverlay);
+        LinearLayout contentBox = (LinearLayout)findViewById(R.id.ICAContentBox);
+
+        TextView title = (TextView)findViewById(R.id.ICATitle);
+        TextView body = (TextView)findViewById(R.id.ICABody);
+        TextView accept = (TextView)findViewById(R.id.ICAAccept);
+        TextView quit = (TextView)findViewById(R.id.ICAQuit);
+
+        overlay.setVisibility(View.INVISIBLE);
+        contentBox.setVisibility(View.INVISIBLE);
+        title.setVisibility(View.INVISIBLE);
+        body.setVisibility(View.INVISIBLE);
+        accept.setVisibility(View.INVISIBLE);
+        quit.setVisibility(View.INVISIBLE);
```

Comment:
Lines 150-160 and 131-141 are pretty much the same. 
The only difference is the value of the visibility constant.
It would make sense to extract a method that would take
the visibility as argument. What do you think?